server/base/src/main/java/org/apache/accumulo/server/util/Admin.java

```diff
@@ -224,7 +225,7 @@ static class RestoreZooCommand {
   @Parameters(commandNames = "fate",
       commandDescription = "Operations performed on the Manager FaTE system.")
   static class FateOpsCommand {
-    @Parameter(description = "[<txId>...]")
+    @Parameter(description = "[<txId> <txId>...]")
```

Comment:
```suggestion
    @Parameter(description = " <txId> { <txId> ... } ")
```
It looks like for other parameters that accept a list we use this format

Reply:
I was trying to show that the `txId` is not always required.

Reply:
I was trying to base it on `git` sytnax guidelines and actually found them [here](https://github.com/git/git/blob/master/Documentation/CodingGuidelines#L639). After reading through their guidelines, I think I could make the list for 0 or more shorter, like `[<txId>...]` and 1 or more like `<txId>...`.

```
[<patch>...]
   (Zero or more of <patch>.  Note that the dots are inside, not
   outside the brackets.)
```